bigframes/core/groupby/__init__.py

```diff
@@ -109,6 +109,7 @@ def __getitem__(
                 dropna=self._dropna,
             )
 
+    @validations.requires_strict_ordering()
```

Comment:
do we plan to implement peek() in groupby classes later and then suggest that here when strict ordering is disabled?

Reply:
No, unlikely we will add peek() to groupby classes, as you can't really do an efficient limit clause per-group in bigquery.